lib/create.js

```diff
@@ -15,22 +20,26 @@ const templateDir = './functionTemplates/nodejs/';
  * @param {string} functionPath - the path to create the function at
  * @returns {string} - the final name selected for the function
  */
-const create = async function create(functionName, functionPath) {
+const create = async function create(functionName, functionPath, runtime) {
   // removing the '-' from monikers string is required because they
   // don't allow the glue string to be empty.
   const finalName = functionName;
   validateName(finalName);
+  const template = runtimeToTemplateDir[runtime];
+  if (!template) {
+    throw new Error(`No template for runtime: ${runtime}`);
+  }
   // parse the templated yml and make the necessary modifications
-  const templatePath = path.join(__dirname, templateDir);
+  const templatePath = path.join(__dirname, templateDir, template);
   const binarisConf = await YMLUtil.loadBinarisConf(templatePath);
```

Comment:
This most likely needs to be changed. I believe this will now expect a separate `binaris.yml` in every template directory (js, py etc) which is redundant but acceptable. But then you add the runtime manually back to the conf which suggests that there is a duplicate `binaris.yml` template in every directory. If each runtime had it's own template dir I would suggest hardcoding, but in lieu of that I would say have them all share a root `binaris.yml` template and update accordingly.